website/docs/documentation/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/README.md

```diff
@@ -0,0 +1,78 @@
+
+| Input name | Input Description | Input Type |
+| --- | --- | --- |
+| fastq1_input_files | Cloud locations for each read1 file | Array of strings | 
+| fastq2_input_files | Optional cloud locations for each read2 file if running paired-end samples |Array of strings |
+| input_ids | Unique identifiers or names for each cell; can be a UUID or human-readable name | Array of strings |
+| input_names | Optional unique identifiers/names to further describe each cell. If `input_id` is a UUID, the `input_name` could be used as a human-readable identifier | String |
+| batch_id | Identifier for the batch of multiple samples | String |
+| batch_name | Optional string to describe the batch or biological sample | String |
+| input_name_metadata_field | Optional input describing, when applicable, the metadata field containing the `input_names` | String |
+| input_id_metadata_field | Optional string describing, when applicable, the metadata field containing the `input_ids` | String |
+| `project_id` | Optional project identifier; usually a number | String |
+| `project_name` | Optional project identifier; usually a human-readable name | String |
+| `library` | Optional description of the sequencing method or approach | String |
+| `organ` | Optional description of the organ from which the cells were derived | String |
+| `species` | Optional description of the species from which the cells were derived | String |
+
+
+### Smart-seq2 Multi-Sample Task Summary
+
+| Task name and link to WDL | Description | Software | Tools |
+| --- | --- | --- | --- |
+| [CheckInputs.checkInputArrays](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/CheckInputs.wdl) | Checks the inputs and initiates the per cell processing | Bash | NA | 
+| [single_nucleus_run.SmartSeq2SingleNucleus](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl) | Calls the Smart-seq2 Single Nucleus workflow to perform alignment, deduplication, quality metric calculating, and exon and intron counting. Produces a final matrix (Loom format) with counts and metrics | See the [Smart-seq2 Single Nucleus Overview](../Smart-seq2_Single_Nucleus_Pipeline/) for details about task software | See the [Smart-seq2 Single Nucleus Overview](../Smart-seq2_Single_Nucleus_Pipeline/) for details about task tools |
+| [LoomUtils.AggregateSmartSeq2Loom](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/LoomUtils.wdl) | Aggregates the matrix files (Loom format) for each sample to produce one final Loom output | Python 3 | Custom script: [ss2_loom_merge.py](https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/loom-output/ss2_loom_merge.py) | 
+
+
+### Outputs
+
+The following table lists the outputs of the Multi-snSS2 workflow.
+
+| Output file name | Output Description | Output Type |
+| --- | --- | --- |
+| bam_files | Array of genome-aligned BAM files (one for each sample) generated with Star  | Array [BAM]|
```

Comment:
are "cell" and "sample" interchangeable here? Here we say one per sample and below it says one per cell, not sure if that means the same thing for SS2

Reply:
They are interchangeable but should be consistent in the table. I updated.